avni-server-api/src/main/resources/application.properties

```diff
@@ -69,15 +69,15 @@ spring.jackson.serialization.fail-on-empty-beans=false
 spring.jackson.mapper.accept-case-insensitive-enums=true
 
 # Application
-debug=false
+debug=true
 avni.defaultUserName=${OPENCHS_USER_NAME:admin}
 spring.servlet.multipart.max-file-size=10028KB
 spring.servlet.multipart.max-request-size=10028KB
 
 # Network
 server.tomcat.protocol-header=x-forwarded-proto
 spring.security.require-ssl=true
-server.port=${OPENCHS_SERVER_PORT:8021}
+server.port=${OPENCHS_SERVER_PORT:8080}
```

Comment:
Revert the changes made to port and debug flag.

Reply:
Had to change the port in my local system as 8021 was already occupied. 
Apologies for pushing this change!